libraries/c_sdk/standard/https/test/system/iot_tests_https_system.c

```diff
@@ -107,18 +107,6 @@
     "-----END CERTIFICATE-----"
 #endif /* ifndef IOT_TEST_HTTPS_ROOT_CA */
 
-/**
- * Client certificate and private key configurations.
- *
- * With PKCS #11 provisioning of the internally stored keys these parameters are deprecated.
- */
-#ifndef IOT_TEST_HTTPS_CLIENT_CERTIFICATE
-    #define IOT_TEST_HTTPS_CLIENT_CERTIFICATE    keyCLIENT_CERTIFICATE_PEM  /**< @brief Client certificate configuration. */
```

Comment:
Since this is deprecated, it would be nice to just delete this, to avoid having to refactor this too.

Reply:
What's the context for this being deprecated?

Reply:
@dcgaws I will ask Sarena